storehaus-hbase/src/main/scala/com/twitter/storehaus/hbase/HBaseStore.scala

```diff
@@ -41,7 +41,7 @@ trait HBaseStore {
   protected val futurePool = FuturePool(Executors.newFixedThreadPool(threads))
```

Comment:
we should have a todo to pass in the `FuturePool`. Should not make these internally